chapter2/exercise1.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+
+    displayTriangle = function() {
```

Comment:
przez to, ze nie uzyles "var" albo "let" / "const" zrobiles global pollution i dodales obiekt do window. to rowniez jeden z powodow dla ktorych powinienes uzywac function declarations

Reply:
tutaj omyłkowo pominąłem słowo kluczowe.